practice-two/src/styles/sections/plan.css

```diff
@@ -0,0 +1,175 @@
+.section-plan {
+  background-color: var(--background-color-primary);
+}
+
+.plan-container {
+  padding-top: 52px;
+  padding-bottom: 42px;
+  text-align: center;
+}
+
+.plan-heading {
+  margin-bottom: 20px;
+  font-size: var(--font-size-lg);
+  color: var(--color-text-primary);
+}
+
+.choose-plan {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  align-items: center;
+  gap: 24px 0;
+}
+
+.plan-card-silver {
+  background: linear-gradient(
+    224deg,
+    var(--background-color-card-silver1) 5.13%,
+    var(--background-color-card-silver2) 90.88%);
+  color: var(--color-text-primary);
+}
+
+.plan-card-gold {
+  background-color: var(--background-color-card-gold);
+  color: var(--color-text-secondary);
+}
+
+.plan-card-platinum {
+  background: linear-gradient(
+    223deg,
+    var(--background-color-card-platinum1) 4.85%,
+    var(--background-color-card-platinum2) 96.5%);
+  color: var(--color-text-primary);
+}
+
+.card-heading,
+.plan-duration {
+  font-size: var(--font-size-xsmall);
+  line-height: var(--line-height-2lg);
+  letter-spacing: var(--letter-spacing-md);
+}
+
+.card-heading {
+  font-family: 'lucida grande', sans-serif;
+  font-weight: var(--font-weight-bold);
+  text-transform: uppercase;
+  color: var(--color-text-tertiary);
+}
+
+.plan-card-gold .card-heading {
+  color: var(--color-text-secondary);
+}
+
+.plan-pricing-info {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 7px;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.plan-price {
+  padding-right: 5px;
+  font-size: var(--font-size-2xl);
+  line-height: var(--line-height-2xl);
+  letter-spacing: var(--letter-spacing-xxsmall);
+
+  & .price-symbol {
+    display: inline-block;
+    transform: translateX(-8px);
```

Comment:
Use padding right instead of `display: inline-block` and `transform: translateX(-8px)`